### ISSUE TITLE
Long Term Activity

### ISSUE THREAD

**Issue by willzfarmer** [sourcegraph/thyme#34]
So I'm starting to use this tool on a daily basis to keep track of what I'm working on at any given moment, and my general flow is to start tracking when I get to work, stop when I leave, and then generate the results at home to see how my day went.

I'd love to be able to compare results from day to day and see my trends. Any thoughts on having the ability to compare long-term results?

**Comment by beyang** [comment]
Yes, I've thought about this, as well. Right now, I'm using it to track daily usage, which means I restart the command every day and have it write to a new file. It would be nice if I could point it at a directory and it would write to files in the form `${thyme-dir}/${date}.json`. Then, we could extend the dashboard to read from the files in that directory. What do you think of that?

**Comment by willzfarmer** [comment]
Yeah, that would be great. I was also thinking you could do some cool metrics to see how activity has changed from day to day, or from week to week. There's a ton of cool stuff you could do involving auto-detection of user trends and then tracking how they change over time.

For instance, you could track how much of your time was spent on reddit on average ;)

In general, I do think the current UI is not well designed for long term activity. It's really good at showing details about what you did, but it doesn't make enough use of summary statistics, and (currently) in order to be understood you have to spend a lot of time grokking it, especially if you ran it for a couple hours.
